runtime/compiler/env/JITServerPersistentCHTable.cpp

```diff
@@ -149,9 +141,7 @@ JITServerPersistentCHTable::commitModifications(const std::string &rawData)
          {
          auto classInfo = findClassInfo(flat->_subClasses[i]);
 
-         // For some reason the subclass info is still null sometimes. This may be indicative of a larger problem or it could be harmless.
-         // JITServer TODO: figure out why this is happening
-         //TR_ASSERT(classInfo, "subclass info cannot be null: ensure subclasses are loaded before superclass");
+         TR_ASSERT_FATAL(classInfo, "subclass info cannot be null: ensure subclasses are loaded before superclass");
```

Comment:
Let's make this into a TR_ASSERT temporarily. The right solution is TR_ASSERT_FATAL, but currently there is a known issue which will hit this assert, the server will die and we will not test the right thing. I am hoping to get to this problem soon and then we can put back the ASSERT_FATAL before the final sync between master and jitaas.